tests/test_esri_tin.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2019 Vincent Cloarec (vcloarec at reos dot site)
+*/
+#include "gtest/gtest.h"
+
+
+#include <fstream>
+
+
+//mdal
+#include "mdal.h"
+#include "mdal_testutils.hpp"
+
+TEST( ESRITinTest, loadMesh )
+{
+  std::string path = test_file( "/esri_tin/top/tnxy.adf" );
+  MeshH m = MDAL_LoadMesh( path.c_str() );
+
+  ASSERT_TRUE( nullptr != m );
+
+  int v_count = MDAL_M_vertexCount( m );
+  EXPECT_EQ( v_count, 13 );
+
+  int f_count = MDAL_M_faceCount( m );
+  EXPECT_EQ( f_count, 19 );
+
```

Comment:
please add test for few faces to see if the vertices are correctly assigned. Also add check for bed elevation dataset